packages/host/src/AppDrawer.tsx

```diff
@@ -45,7 +45,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   drawerMenuItem: {
     height: 32,
-    margin: '20px 0',
+    margin: '16px 0',
```

Comment:
tweaked these styles to make the left navigation nicer when opening and closing the drawer